packages/algob/README.md

```diff
@@ -2,13 +2,29 @@
 
 Algob is the CLI (Command Line Interface) for Algo Builder. Think about it as an Ethereum Truffle but for Algorand projects.
 
+## Packages
+
+### `algob`
+
+This package provides a variety of high level tools like, `executeTransaction`, `mkTxParams`, etc. These high level tools help in making the development phase of algorand applications easier by acting as a middleware between the developer and `algosdk`.
+
+### `runtime`
+
+This package is basically a light version of Algorand runtime. It is efficient in assisting with the tasks of processing smart contract logic i.e. TEAL files as well as in executing transactions. The package also provides various types of parsers like `addressToPk`, `stringToBytes`, `loadFromYamlFileSilent`, etc.
+
 Please read the main [README](https://github.com/scale-it/algo-builder/blob/master/README.md) file for details about the project and `algob`. This file provides a development description about the `algob` package.
 
 ## How to use the API docs?
 
 The best way to find a type, function or something related to a use case is by using the **search field** at the **top** of this page.
 
 
+## Important links
+
++ [Home Page](file:///home/manofcode/scaleit/algo-builder/docs/index.html)
++ [Runtime Docs](file:///home/manofcode/scaleit/algo-builder/docs/api/runtime/index.html)
+
```

Comment:
```suggestion
```

Reply:
why do we have your username here ?

Reply:
i don't think this link will work on other system

Reply:
This link will be changed while deploying. It will contain the actual address after landing page is deployed. I'll make these empty brackets.

Reply:
No. This should have valid links. It won't "change" while deploying. You can use relative links to work. It's just that it currently won't work as master does not have generated API docs. You can update the links like:
```
[Home Page](https://scale-it.github.io/algo-builder)
[Runtime Docs](../runtime/index)
```

Reply:
Please also add one more link for user-docs (`/guide/`)